app/Helpers/database/game.php

```diff
@@ -85,88 +84,103 @@ function getGameMetadata(
     $metricsJoin = '';
     $metricsBindings = [];
     if ($metrics) {
-        $cacheKey = "game:$gameID:playercount";
-        $gameMetrics = Cache::get($cacheKey);
-        if ($gameMetrics === null) {
+        if (env('FEATURE_AGGREGATE_QUERIES')) {
             $parentGameId = getParentGameIdFromGameTitle($gameDataOut['Title'], $gameDataOut['ConsoleID']);
 
-            $bindings = [
-                'gameId' => $gameID,
-                'achievementFlag' => $flag,
-            ];
-
-            $gameIdStatement = 'ach.GameID = :gameId';
-            if ($parentGameId !== null) {
+            if ($parentGameId === null) {
+                $query = "SELECT players_total AS NumDistinctPlayers FROM GameData WHERE ID=$gameID";
+                $gameMetrics = legacyDbFetch($query);
```

Comment:
**game/1446**
With feature: 0ms
Without feature: 743ms (results are cached for one hour at a time)

NOTE: due to the data available in the aggregate columns, only `NumDistinctPlayers` is available. Previously, this function returned both `NumDistinctPlayersCasual` and `NumDistinctPlayersHardcore`. The game page only used `NumDistinctPlayersCasual`, which reflects both hardcore and non-hardcore players. There are two APIs that also return the data. I'm not sure how that's being used, but I've made the decision to deprecate the fields and return `NumDistinctPlayers` instead. The deprecated fields return the same value as `NumDistinctPlayers`.

Reply:
If I understand correctly I will add `players_total_hardcore` to the games table. This was an oversight on my part.

Reply:
Makes sense. It might be interesting to know for reporting? To see how sets fare when it comes to opting for softcore, might be an indicator of a set's difficulty in the future?

Reply:
I'm absolutely fine with deprecating the distinction btw.